code/isabel/Python/labs/api_requests/dad_joke_part2.py

```diff
@@ -0,0 +1,22 @@
+import requests
```

Comment:
Same as Weather API, I want you to reach into the JSON file and only print the jokes like you did for part one.